packages/lesswrong/components/common/SearchBar.jsx

```diff
@@ -69,78 +102,34 @@ class SearchBar extends Component {
   }
 
   render() {
-    const inputOpenClass = this.state.inputOpen ? "open" : null;
-    const resultsOpenClass = this.state.searchOpen ? "open" : null;
     const algoliaAppId = getSetting('algolia.appId')
     const algoliaSearchKey = getSetting('algolia.searchKey')
 
+    const { classes } = this.props
     const { searchOpen } = this.state
 
     if(!algoliaAppId) {
-      return <div className="search-bar">Search is disabled (Algolia App ID not configured on server)</div>
+      return <div className={classes.root}>Search is disabled (Algolia App ID not configured on server)</div>
```

Comment:
This should be `classes.searchBar`, though actually I think the `searchBar` class should be renamed to `root` to comply with the Material UI conventions.

Reply:
Not 100% sure about this (I'm not sure it's technically the root element), but went ahead and changed it to root